actionview/lib/action_view/helpers/date_helper.rb

```diff
@@ -687,6 +684,16 @@ def time_tag(date_or_time, *args, &block)
 
         content_tag("time".freeze, content, options.reverse_merge(datetime: datetime), &block)
       end
+
+      def normalize_distance_of_time_argument_to_time(value)
```

Comment:
This needs to be private so you'll need to add `private` and then indent again or rubocop will complain